### ISSUE TITLE
Windows 10 / 11 Security detects that Freewallet is a Trojan.

### ISSUE THREAD

**Issue by zokunfts** [jdogresorg/freewallet-desktop#109]
It is not a friendly letter of an introduction than the Wallet where you are going to operate and store a large number of your precious memes and tokens, and activate the Windows alarms. Perhaps it is worth thinking about why the Windows alarm is activated and solving it to improve the landing of new users.

![image_2022-12-12_17-15-28](https://user-images.githubusercontent.com/107966598/207160075-ec1f7e97-0bc1-46ea-8885-f96dda92e085.png)

**Comment by Hij34** [comment]
Good

**Comment by cryptoanedrico** [comment]
Please address this.

**Comment by davestaxcp** [comment]
This is very important to be addressed and explained.

**Comment by jdogresorg** [comment]
I get no such alarm when installing Freewallet on windows 10/11.

**Comment by davestaxcp** [comment]
This issue should not be closed.... and I cannot come up with why you dont receive this alarm on your system, @jdogresorg 

Not only does Windows see Freewallet is from an Unknown Publisher:

![image](https://github.com/jdogresorg/freewallet-desktop/assets/136373423/2497edad-d3b7-42de-bd39-1d995206557c)

When you install it, Windows says that in the Freewallet files that "/SPAWNWAND=$403D2 /NOTIFYWND=$40418" is a trojan....

**Comment by jdogresorg** [comment]
Please post screenshots of this error message... I am not getting any error like this when installing freewallet, and other than a couple complaints here, not hearing of this issue from the larger community.

**Comment by jdogresorg** [comment]
will try a clean install on new windows 11 machine and see if I get any trojan error... will leave issue open until I am able to confirm I am not getting Trojan warning on fresh windows 11 install

**Comment by davestaxcp** [comment]
OK, I can see how I wrongly assumed that an "unidentified developer" = trojan and Windows stating:

![image](https://github.com/jdogresorg/freewallet-desktop/assets/136373423/aef522cc-d05a-4791-b35e-21365e705f59)

and I was wrong in assumption that this program = a trojan

- my worry here is that any new user will interpret all of these things as a risk to their privacy and computer.

Is there a way you can be into contact with Windows to show "Jeremy Johnson" or "Freewallet.io" as the publisher? So the user doesn't have an inkling of feeling like they are at risk?

I hope you can at least understand a user seeing the words "Do you want to allow this app from an unknown publisher to make changes to your device" turns off new users who dont know your background, nor the open source situation that Freewallet derives from?

**Comment by jdogresorg** [comment]
yes, signing the app would be ideal, which is why there is an open issue for this... just need to find the time to look into it and integrate it into the windows build process.

**Comment by davestaxcp** [comment]
ok
